manual/manual_circuit.rst

```diff
@@ -985,7 +985,7 @@ Though less expressive than native dictionary serialization, it is widely suppor
 
 .. Quipper
 
-.. note:: The OpenQASM converters do not support circuits with `implicit qubit permutations <https://cqcl.github.io/pytket/manual/manual_circuit.html#implicit-qubit-permutations>`_ . This means that if a circuit contains such a permutation it will be ignored when exported to OpenQASM format.
+.. note:: The OpenQASM converters do not support circuits with `implicit qubit permutations <https://cqcl.github.io/pytket/manual/manual_circuit.html#id9>`_ . This means that if a circuit contains such a permutation it will be ignored when exported to OpenQASM format.
```

Comment:
Can we have this as an internal link, which sphinx can then translate to an html link? We don't want to have to check generated tags when updating rst files.

Reply:
Yes this would be better. I've done it now.

Strangely this requries a separate sphinx extension.
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html

There are also now warninging about duplcate section headings for example "advanced topics" appears in multiple places. I assume these warnings are to ensure that the heading linked to is unique.

Reply:
I can modify the manual sections to avoid these warnings "advanced topics" -> "advanced circuit topics" or "advnaced circuit construction topics" if preferred.

Reply:
Sounds worth doing to get rid of the warnings (which might actually be important for correct hyperlinking).

Reply:
Okay, I've removed the duplicate headings